src/actions/common.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
Remove.

Reply:
This is from the app template... There are a couple of `console.error()` statements in this file.